pkg/sti/script/installer.go

```diff
@@ -108,6 +109,15 @@ func (s *handler) download(scripts []api.Script, workingDir string) (bool, error
 		}
 	}
 
+	// check for the scripts inside the sources
+	scriptsDir := filepath.Join(workingDir, "/upload/src/.sti/bin")
+	if s.fs.Exists(scriptsDir) {
+		for _, script := range scripts {
+			downloads[script] <- true
```

Comment:
don't you need to actually check that each of these files exists, not just the .sti/bin dir, when setting download to true?